src/printer.js

```diff
@@ -62,7 +62,7 @@ const {
 function shouldPrintComma(options, level) {
   level = level || "none";
 
-  switch (options.trailingComma) {
+  switch (options.trailingCommaPHP) {
```

Comment:
Let's support also `options.trailingComma` for `all` and `none` for better integration